recipes/regain/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "regain" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 03a83d3c58c36f71cdf2be5a987630a0a603c190a7acc62a085bb45deca2699e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+
+  run:
+    - python >=3.6
+    - numpy >=1.8.2
+    - scikit-learn >=0.17
+
+test:
+  imports:
+    - regain
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/fdtomasi/regain
+  summary: "REGAIN (Regularised Graphical Inference)."
+  description: |
+    Regularised graph inference across multiple time stamps, considering the influence of latent variables. 
+    It inherits functionalities from the scikit-learn package.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  doc_url: https://github.com/fdtomasi/regain
+  dev_url: https://github.com/fdtomasi/regain
+
+extra:
+  recipe-maintainers:
+    - fdtomasi
+    - veronicatozzo
+    - samuelefiorini
```

Comment:
Are these people happy to be maintainers of this `conda-forge` package? Do you not want to add yourself here @Vlasovets?

![image](https://user-images.githubusercontent.com/881019/184103819-843a3ab2-200e-453a-b9ce-65b0eff720e2.png)

Reply:
hi, @dhirschfeld!
and thanks for reviewing my recipe!
I'm in touch with Federici Tomasi (fdtomasi) who has develop this library, and I've just created a recipe to publish it on conda-forge, so I can also publish my own library which depends on this one.
I guess you are right that I should be the maintainer of the recipe, I will change it, thank you!
About the LICENSE.txt, it is exactly the same license agreement from the main [library](https://github.com/fdtomasi/regain/blob/master/LICENSE), what do you mean by restoring it? 
If I remove it from the directory then the test will assert me of not having a licence file.

Reply:
After the feedstock is created you can reach out and see if anyone else wants to be added as a maintainer and use the bot command to do so:
https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-user-username

Reply:
@dhirschfeld, thanks again for the valuable feedback!
@conda-forge-admin, may I ask you to take a look at my PR, please?